utils/onClickEyeIcon.js

```diff
@@ -0,0 +1,27 @@
+// const
+const EYE_ICON_OFF = "icon-eye-off";
+const EYE_ICON_ON = "icon-eye-on";
+
+// DOM
+const passwordInputList = [
+  ...document.querySelectorAll("input[type='password']"),
+];
+const eyeIconList = [...document.querySelectorAll(".password-box i")];
+
+function handleClickEyeIcon({ target }) {
+  if (target.tagName !== "I") {
+    return;
+  }
+  const inputIndex = eyeIconList.indexOf(target);
+  if (target.classList.contains(EYE_ICON_OFF)) {
```

Comment:
classList로 분기를 태우는 것보다는, 실제 주어는 passwordInput DOM 의 type이기 때문에 이렇게 분기를 태우는건 어떨까요??
```suggestion
  if (passwordInputList[inputIndex].type = "password") {
```